WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1347,10 +1347,10 @@ private void performPrimaryAction() {
             case PUBLISH_NOW:
             case UPDATE:
             case SCHEDULE:
+            case SUBMIT_FOR_REVIEW:
```

Comment:
I might have asked this already, but I forgot the explanation, sorry.

Is it okay that we track `EDITOR_POST_PUBLISH_TAPPED` for all the actions (not just "publish")?

Reply:
This is a good observation. Thanks!

Reply:
f4a0fcd

Reply:
Follow up question :D - is it okay that we don't track events for the remaining buttons?

Reply:
Thanks. They should be tracked as well. 🙏